src/app/component/alumni-form/alumni-form.component.ts

```diff
@@ -18,116 +18,22 @@ import { FormDataa } from 'src/app/FormDataa';
 })
 export class AlumniFormComponent implements OnInit {
   alumniFeedback: FormGroup; //step 1
-  dataAlumni: FormDataa;
```

Comment:
You did something good here. Idk why I didn't think of making it an array.hehe :)